guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -44,9 +50,8 @@ This is why we recommend migrating before the upgrade, at which point you can ea
 [id="katello-deprecations"]
 == Deprecations
 
-=== The deprecated `auto_attach` and `autoheal` parameters have been removed from all API endpoints.
```

Comment:
I thought that this was by design. It's just a long H3 heading as far as I can tell: https://docs.theforeman.org/3.17/Release_Notes/index-katello.html#katello-deprecations

same line 56.